Tests/Reqnroll.GeneratorTests/VirtualFileSystem.cs

```diff
@@ -214,5 +214,10 @@ static void CheckSearchPattern(string searchPattern)
                 throw new ArgumentException("Illegal characters in path", "searchPattern");
             }
         }
+
+        public void CreateDirectory(string path)
+        {
+            throw new NotImplementedException();
```

Comment:
All other methods are implemented. Isnt this confusing?

Reply:
Yes. I can see how that can be confusing. 

This file seems to be a special stub implementation of IFilesystem for the purposes of the generator tests. None of the tests use the CreateDirectory function I've added to the interface. So this 'not implemented' is simply a place holder.

Reply:
Added in an implementation that was missing.